swap/prices.go

```diff
@@ -33,9 +33,15 @@ allowing for a multi-currency design.
 */
 
 // Placeholder prices
+// Based on a very crude calculation: average monthly cost for bandwidth in the US / average monthly bill for bandwidth in the US
+// 67GB / $190 = $0.35 / GB
+// 0.35 / (1000000000 * 4096) = $0.0000014336 / chunk
```

Comment:
Minor suggestion, for readability I would short the numbers as 10^6

Reply:
Done

Reply:
Done